omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/PersonResource1_8.java

```diff
@@ -391,7 +394,8 @@ public static PersonAddress getPreferredAddress(Person instance) {
 	 */
 	@Override
 	public Person getByUniqueId(String uuid) {
-		return Context.getPersonService().getPersonByUuid(uuid);
+		this.person = Context.getPersonService().getPersonByUuid(uuid);
```

Comment:
Why did you change this?

Reply:
I have presented my thought on this in the comment on the `Person` class level variable.